src/phantom/base/typography/_bundle.scss

```diff
@@ -2,3 +2,15 @@
 @import "links_config";
 @import "lists_config";
 @import "paragraphs_config";
+
+@font-face {
+  font-family: "Calibre Light";
+  src: url('https://casper.com/static-assets/nightshade-core/src/typography/fonts/CalibreWeb-Light.woff2') format('woff2'),
```

Comment:
I think we're better off not including these in the asset pipeline on either service in that case, I think the best approach is a new bucket in the _new_ AWS account, where we link to those fonts from each app. Thoughts?